### ISSUE TITLE
SliderInt support for stb_sprintf() comma value

### ISSUE THREAD

**Issue by metarutaiga** [ocornut/imgui#3604]
**Version/Branch of Dear ImGui:**

Version: 1.79
Branch: docking
Define: IMGUI_USE_STB_SPRINTF

**Back-end/Renderer/Compiler/OS**

Back-ends: imgui_impl_osx.mm + imgui_impl_metal.mm
Compiler: Apple clang version 12.0.0 (clang-1200.0.32.27)
Operating System: macOS 11.0.1

**My Issue/Question:**

I try to print a comma value, but it would be get wrong value when sliding.
It seem a problem in ImAtoi that don't support comma.

**Screenshots/Video**

![A](https://user-images.githubusercontent.com/52752725/99618919-4ef5f880-2a5d-11eb-8b4d-93cbb31e763a.png)
![B](https://user-images.githubusercontent.com/52752725/99618924-51585280-2a5d-11eb-8fda-81b229fdce64.png)

**Standalone, minimal, complete and verifiable example:** 

```
ImGui::SliderInt("Value", &value, 100000, 10000000, "%'d");
```

**Comment by metarutaiga** [comment]
In addition, the ' flag in printf is the standard POSIX 2008.
In my test, MSVC is still unsupported it now.

[https://stackoverflow.com/questions/1449805/how-to-format-a-number-from-1123456789-to-1-123-456-789-in-c](url)

**Comment by ocornut** [comment]
This has now been fixed by 8ee77f1b65cd8f1c6217f62c90372041be1c5529 and e842b196df8289f78356a5a571d5b914e1461f98.
